docs/authentication/configuration/restrictions.mdx

```diff
@@ -0,0 +1,55 @@
+---
+sanity_slug: /docs/authentication/restrictions
+---
+
+# Overview
+
+# Allowlist
+
+The Allowlist feature allows you to control who can get access to your application. It can restrict sign-ups to only a certain set of email addresses or phone numbers that you define. This can be used in a way that acts as a reverse blocklist, only allowing the users added to access your application and blocking all others. (Note: Allowlist is a premium feature and is not available on the Free plan. [Upgrade your plan](https://clerk.com/pricing) to enable this feature.)
```

Comment:
```suggestion
The Allowlist feature allows you to control who can get access to your application. It can restrict sign-ups to only a certain set of email addresses or phone numbers that you define. This can be used in a way that acts as a reverse blocklist, only allowing the users added to access your application and blocking all others.
```